packages/shared/src/components/squads/Details.tsx

```diff
@@ -206,150 +164,109 @@ export function SquadDetails({
           </SquadSubTitle>
         </div>
       )}
-      <div className={classNames('flex flex-col', className)}>
-        <form
-          className="-mb-2 flex flex-col items-center gap-4"
-          onSubmit={handleSubmit}
-          onBlur={handleChange}
-          id="squad-form"
-        >
-          {!createMode && (
-            <ImageInput
-              initialValue={form.image ?? form.file}
-              id={squadImageId}
-              fallbackImage={cloudinary.squads.imageFallback}
-              className={{
-                container: 'my-1 !rounded-full border-0',
-                img: 'object-cover',
-              }}
-              hoverIcon={<CameraIcon size={IconSize.Large} />}
-              alwaysShowHover={!imageChanged}
-              onChange={() => setImageChanged(true)}
-              size="large"
-            />
-          )}
-          <div className="flex w-full max-w-lg flex-col justify-center gap-4">
-            <TextField
-              label={createMode ? 'Name your Squad' : 'Squad name'}
-              inputId="name"
-              name="name"
-              valid={!!name}
-              leftIcon={<SquadIcon />}
-              value={name ?? ''}
-              className={{
-                container: 'w-full',
-              }}
+      <form
```

Comment:
Most of the changes here is the wrapper. Maybe I should put them in a separate commit next time.

Reply:
just to check that this is tested on all screens with all the changes and it still works ok?

Reply:
Yep. The screenshots I attached were taken after the changes.